azure-container-apps-maven-plugin/pom.xml

```diff
@@ -0,0 +1,191 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.microsoft.azure</groupId>
+        <artifactId>azure-maven-plugins</artifactId>
+        <version>1.40.0-SNAPSHOT</version>
+    </parent>
+
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>azure-container-apps-maven-plugin</artifactId>
```

Comment:
We may omit the leading `azure-` given the group ID already have the `.azure` part. Check other plugins for consistency.